bootstrap.sh

```diff
@@ -28,6 +28,9 @@ echo -e "👉 Create symlink for pre-commit hooks"
 # Symlink needs to be ../../hooks and not ./hooks because git evaluates them in .git/hooks
 ln -sf ../../hooks/pre-commit.sh .git/hooks/pre-commit
 chmod +x .git/hooks/pre-commit
+ln -sf ../../hooks/post-rewrite.sh .git/hooks/post-rewrite
```

Comment:
When the hooks are exactly the same, why we don't do just:
```
ln -sf ../../hooks/pre-commit.sh .git/hooks/post-rewrite
```

Reply:
Makes sense... changed and deleted the file...